src/base/bittorrent/abstractfilestorage.cpp

```diff
@@ -53,7 +55,7 @@ void BitTorrent::AbstractFileStorage::renameFile(const Path &oldPath, const Path
         if ((renamingFileIndex < 0) && (path == oldPath))
             renamingFileIndex = i;
         else if (path == newPath)
-            throw RuntimeError(tr("The file already exists: '%1'.").arg(newPath.toString()));
+            throw RuntimeError(tr("The file already exists in this torrent: '%1'.").arg(newPath.toString()));
```

Comment:
This message is more confusing. There is no indication of what torrent it is referring to.